.travis.yml

```diff
@@ -10,15 +10,13 @@ php:
   - 7.0
   - 7.1
 
-  matrix:
+matrix:
   fast_finish: true
   include:
```

Comment:
This brings in the current version of HHVM, 3.17, however is still throwing the same error with 
```This package requires php ^7.0 but your HHVM version does not satisfy that requirement.```
This error is also happening on other repos like the primary doctrine2 repo per this test - https://travis-ci.org/doctrine/doctrine2/jobs/188191005

Not sure what else I can do with this as HHVM 3.17 is not reporting to be PHP 7.x compatible per composer.